go.mod

```diff
@@ -1,12 +1,10 @@
 module github.com/PaddleHQ/go-aws-ssm
 
-go 1.12
+go 1.17
```

Comment:
This change will force everyone that uses the library to bump to 1.17 Go version, is it possible to still keep 1.12 here?

Reply:
hey, sure I'll check v1.12 compatibility.